lib/glific_web/providers/maytapi/shunt.ex

```diff
@@ -60,6 +60,21 @@ defmodule GlificWeb.Providers.Maytapi.Plugs.Shunt do
     |> Router.call(opts)
   end
 
+  @doc false
+  def call(%Conn{params: %{"status" => _status}} = conn, opts) do
```

Comment:
Can we add a `%{"type" => "status"}` to the pattern matching map here, since we should differentiate by the type of the messages first..